prowler/providers/aws/services/ec2/ec2_securitygroup_allow_ingress_from_internet_to_port_mongodb_27017_27018/ec2_securitygroup_allow_ingress_from_internet_to_port_mongodb_27017_27018.py

```diff
@@ -1,30 +1,36 @@
 from prowler.lib.check.models import Check, Check_Report_AWS
 from prowler.providers.aws.services.ec2.ec2_client import ec2_client
 from prowler.providers.aws.services.ec2.lib.security_groups import check_security_group
+from prowler.providers.aws.services.vpc.vpc_client import vpc_client
 
 
 class ec2_securitygroup_allow_ingress_from_internet_to_port_mongodb_27017_27018(Check):
     def execute(self):
         findings = []
         check_ports = [27017, 27018]
         for security_group in ec2_client.security_groups:
-            report = Check_Report_AWS(self.metadata())
-            report.region = security_group.region
-            report.resource_details = security_group.name
-            report.resource_id = security_group.id
-            report.resource_arn = security_group.arn
-            report.resource_tags = security_group.tags
-            report.status = "PASS"
-            report.status_extended = f"Security group {security_group.name} ({security_group.id}) does not have MongoDB ports 27017 and 27018 open to the Internet."
-            if not security_group.public_ports:
-                # Loop through every security group's ingress rule and check it
-                for ingress_rule in security_group.ingress_rules:
-                    if check_security_group(
-                        ingress_rule, "tcp", check_ports, any_address=True
-                    ):
-                        report.status = "FAIL"
-                        report.status_extended = f"Security group {security_group.name} ({security_group.id}) has MongoDB ports 27017 and 27018 open to the Internet."
-                        break
-            findings.append(report)
+            # Check if ignoring flag is set and if the VPC and the SG is in use
+            if not ec2_client.audit_info.ignore_unused_services or (
+                vpc_client.vpcs[security_group.vpc_id].in_use
+                and len(security_group.network_interfaces) == 0
+            ):
```

Comment:
Just to understand, the code within the if gets executed if:
- `ignore_unused_services` is set to `False`
or 
- The VPC is in use and the `security_group.network_interfaces` is 0? If the security group has no interfaces attached it should be not in use, don't?

Reply:
You're right, let me change it to `> 0`